v22.2/show-backup.md

```diff
@@ -19,7 +19,19 @@ For guidance on the syntax for `SHOW BACKUP FROM`, see the [examples](#examples)
 
 `SHOW BACKUP` requires read permissions to its target destination.
 
-{% include {{ page.version.version }}/misc/non-http-source-privileges.md %}
+The user that runs `SHOW BACKUP` to external storage does **not** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
+
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
+
+The user that runs `SHOW BACKUP` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```

Comment:
Same as the comment in destination-privileges.md I don't know if we want to give the user the impression that there are still operations that only admin can run, because thats not the case anymore. EXTERNALIOIMPLICITACCESS governs who can and cannot run against an ExternalStorage.